samples/samples/snippets.py

```diff
@@ -3017,6 +3017,58 @@ def directed_read_options(
     # [END spanner_directed_read]
 
 
+# [START spanner_create_instance_with_autoscaling_config]
+def create_instance_with_autoscaling_config(instance_id):
+    """Creates a Cloud Spanner instance with an autoscaling configuration."""
+    from google.cloud.spanner_admin_instance_v1.types import \
+        spanner_instance_admin
+
+    spanner_client = spanner.Client()
+
+    config_name = "{}/instanceConfigs/regional-us-central1".format(
+        spanner_client.project_name
+    )
+
+    autoscaling_config = spanner_instance_admin.AutoscalingConfig(
+        autoscaling_limits=spanner_instance_admin.AutoscalingConfig.AutoscalingLimits(
+            min_nodes=1,
+            max_nodes=2,
+        ),
+        autoscaling_targets=spanner_instance_admin.AutoscalingConfig.AutoscalingTargets(
+            high_priority_cpu_utilization_percent=65,
+            storage_utilization_percent=95,
+        ),
+    )
+
+    request = spanner_instance_admin.CreateInstanceRequest(
```

Comment:
Can we add a comment here
`When autoscaling_config is enabled, node_count and processing_units fields need not be specified. `